examples/complete/main.tf

```diff
@@ -2,37 +2,45 @@ provider "aws" {
   region = var.region
 }
 
+locals {
+  enabled = module.this.enabled
+}
+
 module "vpc" {
   source  = "cloudposse/vpc/aws"
-  version = "0.28.1"
+  version = "2.0.0"
 
-  cidr_block = var.vpc_cidr_block
+  ipv4_primary_cidr_block = var.vpc_cidr_block
 
   context = module.this.context
 }
 
 module "subnets" {
   source  = "cloudposse/dynamic-subnets/aws"
-  version = "0.39.8"
+  version = "2.1.0"
 
   availability_zones   = var.availability_zones
   vpc_id               = module.vpc.vpc_id
-  igw_id               = module.vpc.igw_id
-  cidr_block           = module.vpc.vpc_cidr_block
-  nat_gateway_enabled  = true
+  igw_id               = [module.vpc.igw_id]
+  ipv4_cidr_block      = [module.vpc.vpc_cidr_block]
+  nat_gateway_enabled  = false
   nat_instance_enabled = false
 
   context = module.this.context
 }
 
 resource "aws_ecs_cluster" "default" {
-  name = module.this.id
-  tags = module.this.tags
+  #bridgecrew:skip=BC_AWS_LOGGING_11: not required for testing
+  count = local.enabled ? 1 : 0
```

Comment:
what's the point for this in an example? 🤔

Reply:
You need to ask cloudposse about that ^^